allofplos/gdrive.py

```diff
@@ -0,0 +1,160 @@
+import datetime
+import os
+import tarfile
+from zipfile import ZipFile, BadZipFile
+
+import requests
+from tqdm import tqdm
+
+from . import get_corpus_dir
+
+# Variables needed
+zip_id = '0B_JDnoghFeEKLTlJT09IckMwOFk'
+metadata_id = '0B_JDnoghFeEKQUhKWXBOVy1aTlU'
+local_zip = 'allofplos_xml.zip'
+zip_metadata = 'zip_info.txt'
+time_formatting = "%Y_%b_%d_%Hh%Mm%Ss"
```

Comment:
Is this the same time_formatting you use elsewhere? is anything else expecting this time_formatting to be followed?

Reply:
@mpacer this is how times are used for the zip file naming in the private repo. Do you have a preferred format? Not used anywhere else for e.g. pubdates, which are only year month and day